Formula/pymol.rb

```diff
@@ -0,0 +1,88 @@
+class Pymol < Formula
+  include Language::Python::Virtualenv
+  desc "Open-source pymol ver. 2.2.0"
+  homepage "https://pymol.org/2/"
+  url "https://github.com/schrodinger/pymol-open-source/archive/v2.2.0.tar.gz"
+  sha256 "58d910103dc494c49c86bc8fd6cd94b1a030647f9d72f69fbd7d7ad25fb11233"
+
+  depends_on "freeglut"
+  depends_on "freetype"
+  depends_on "glew"
+  depends_on "glm" # PyMOL >= 2.2
+  depends_on "libpng"
+  depends_on :macos => :sierra
```

Comment:
Please change this line to `depends_on :macos` and move it above the other dependencies.

Reply:
Thank you. I really appreciate your help. The formula finally works well!!
The dependency `:macos` will be removed if the `pulseaudio` problem is solved.